javascript_translations.php

```diff
@@ -15,7 +15,7 @@
 # along with MantisBT.  If not, see <http://www.gnu.org/licenses/>.
 
 /**
- * Handle langauge translations for Javascript
+ * Handle language translations for Javascript
```

Comment:
Remove this change, there is no point in patching comments in a 3rd party library